modules/network-observability-multitenancy.adoc

```diff
@@ -5,15 +5,17 @@
 :_mod-docs-content-type: PROCEDURE
 [id="network-observability-multi-tenancy{context}"]
 = Enabling multi-tenancy in Network Observability
-Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project admins. Project admins who have limited access to some namespaces can access flows for only those namespaces.
+Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project administrators. Project administrators who have limited access to some namespaces can access flows for only those namespaces. 
+
+For Developers, multi-tenancy is available for both Loki and Prometheus but requires different access rights.
 
 .Prerequisite
 * You have installed at least link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7]
-* You must be logged in as a project administrator
+* You must be logged in as a project administrator or developer
```

Comment:
actually, they must be logged in as cluster admins. It's admins who can grant permissions to less privileged users